### ISSUE TITLE
update Lmod cache for `generic` targets

### ISSUE THREAD

**Issue by trz42** [NorESSI/software-layer#82]
PRs #80 and #81 have been built in parallel. After both have been ingested the Lmod cache needs to be updated or subsequent PRs may unnecessarily rebuild existing packages.

**Comment by trz42** [comment]
- [ ] the builds for all required architectures succeeded (SUCCESS message + reasonably sized tarball)
  - builds for other architectures than `generic` are ignored
  - [ ] `x86_64`
  - [ ] `aarch64`
- [ ] the PR up-to-date with the target branch `nessi.no-2022.11-generic-rebuild` in the repository (if not what are the differences)
- [ ] all changes are sound (checking files changed on GitHub.com)
- [ ] the easyconfig/s being built is/are included with the EB version used (if not why not)
  - no easyconfig was built
- [ ] no changes were needed to get the build(s) succeed (common changes for all architectures, changes for a single architecture, changes because of build environment specifics, etc.)

**Comment by trz42** [comment]
Already done in #83
